centinel/primitives/traceroute.py

```diff
@@ -28,6 +28,10 @@ def traceroute(domain, method="udp", cmd_arguments=[], external=None):
     # the method specified by the function parameter here will
     # over-ride the ones given in cmd_arguments because
     # traceroute will use the last one in the argument list.
+
+    if cmd_arguments is None:
```

Comment:
Interesting. I didn't know that python initialized parameters with the same scope as the function declaration, not within the scope of the function execution.